consensus/src/firststep/handler.rs

```diff
@@ -106,65 +106,95 @@ impl<D: Database> MsgHandler<Message> for Reduction<D> {
         step: u8,
         committee: &Committee,
     ) -> Result<HandleMsgOutput, ConsensusError> {
+        assert_eq!(step, self.curr_step, "msg does not belong to curr step");
```

Comment:
isn't this `assert_eq` too much risky? Would it be possible to handle the error?

Reply:
It's a leftover, I'll change to an err log entry

Reply:
Updated